app/api/ada/adaAddress.js

```diff
@@ -85,8 +85,8 @@ export function getAdaAddressesByType(addressType: AddressType): Promise<AdaAddr
  */
 export async function popBip44Address(type: AddressType): Promise<AdaAddress> {
   return type === 'Internal'
-    ? popBip44InternalAddress()
-    : popBip44ExternalAddress();
+    ? await popBip44InternalAddress()
+    : await popBip44ExternalAddress();
```

Comment:
Should it be `await` actually? Cuz function itself returns `Promise`

Reply:
In this case it actually doesn't matter if you write async or not